crates/rolldown/src/ast_scanner/impl_visit.rs

```diff
@@ -177,7 +189,21 @@ impl<'me, 'ast> Visit<'ast> for AstScanner<'me> {
           request.value.as_str(),
           ImportKind::Require,
           request.span().start,
-          request.span().is_empty(),
+          if request.span().is_empty() {
+            ImportRecordMeta::IS_UNSPANNED_IMPORT
+          } else {
+            let mut meta = ImportRecordMeta::empty();
+            if self
+              .nearest_parent_stmt_info
+              .as_ref()
+              .map(|stmt_info| stmt_info.span_start == expr.span().start && stmt_info.is_call_expr)
```

Comment:
I wonder if we could use oxc's API to check if a `expression` get used.

And it seems the "used" concept is quite complex

Some cases I thought of
```js
require(...); // Not used
(1, require(...)); // Not used

var a = (1, require(...)); // Used
require(...).default; // Used
var a = require(...); // Used
console.log(require(...)); // Used

function foo() {
  return require(...) // Used
}
```

Reply:
Could you add a test case using the above code. Maybe under `crates/rolldown/tests/topics/unused_require`?

Reply:
added